tests/test_conduit_cli.py

```diff
@@ -37,7 +37,7 @@ def test_create_task(subprocess_call):
     create_task("dbt-app", "staging", "arn::blah")
 
     subprocess_call.assert_called_once_with(
-        "copilot task run -n dbtunnel --dockerfile Dockerfile --secrets DB_SECRET=arn::blah --app dbt-app --env staging",
+        "copilot task run -n dbtunnel --image public.ecr.aws/uktrade/tunnel --secrets DB_SECRET=arn::blah --app dbt-app --env staging",
```

Comment:
I think we can make this a constant in `conftest.py`, or... is that not necessary?

We could also link the two together, so one constant used in all places.

Reply:
I didn't want the constants linked because I'd like a change to that constant to mean that a unit test fails

Reply:
That sounds fair to me.